src/python/nxt_python_asgi.c

```diff
@@ -603,6 +604,27 @@ nxt_py_asgi_create_http_scope(nxt_unit_request_info_t *req)
         return NULL;
     }
 
+    if (prefix.length > 0) {
+        i = prefix.length;
+        target_length = r->path_length;
+        p = nxt_unit_sptr_get(&r->path);
+        target = (char*) prefix.start;
+        if (
+            (target_length >= i && nxt_strncmp(target, p, i) == 0) ||
+            // Request path matches prefix but doesn't end with "/"
+            ((target_length + 1) == i && nxt_strncmp(target, p, i - 1) == 0)
+        ) {
+            v = PyString_FromStringAndSize((char*) target, i - 1);
+            if (nxt_slow_path(v == NULL)) {
+                nxt_unit_req_alert(req, "Python failed to create a 'root_path'"
+                                        " string");
```

Comment:
Style issue: when splitting long log message the space should be left at the end of previous line, rather than at the beginning of the next line.